src/Output/JsonCompact.php

```diff
@@ -0,0 +1,70 @@
+<?php
+
+declare(strict_types=1);
+
+namespace SimPod\ClickHouseClient\Output;
+
+use function Safe\json_decode;
+
+final class JsonCompact implements Output
```

Comment:
uniká mi něco nebo je to úplně stejný jako Json?

Reply:
jj jen data chodi bez keys

Reply:
no a proč teda pro tenhle format nepoužiješ json output, když je to stejný?

Reply:
nejsou stejny, data chodi bez keys. jsou to dva ruzny druhy outputu

Reply:
no ... ale implementace je stejná, ne? :D

Reply:
jsou to dva ruzny output formaty v dokumentaci. kdybych nebyl linej a zdokumentoval spravne array shape `data` tak se lisi. ale to az nekdy bude mit sikovnejsi support

Reply:
Mě teda přijde duplikovat kód jen tak pro nic za nic docela zbytečný ...

Reply:
tohle je presne to co pod DRY nespada. to ze ten kod je zrovna stejnej neznamena, ze je to to samy. jsou pripady, kdy je lepsi mit stejnej kod nez se to snazit nejak abstrahovat

tohle je docela dobrej priklad https://overreacted.io/goodbye-clean-code/ i kdyz tam spatne pouziva termin clean code

Reply:
Jejich příklad je dobrej no, tvůj ne :D Důležitý tady je, že kdž budeš potřebovat ty dvě strategie implementovat jinak, tak je to as simple as ctrl+c a ctrl+v, a pak udělání změn ... ale do tý doby máš o soubor míň. Mít duplicitní třídu jenom kvůli jménu prostě nedává smysl